compat/src/render.js

```diff
@@ -104,7 +104,8 @@ options.vnode = vnode => {
 
 	if (type) {
 		// Alias `class` prop to `className` if available
-		if (props.class != props.className) {
+		// only alias on dom elements
+		if (typeof type === 'string' && props.class != props.className) {
```

Comment:
Only aliasing on dom elements breaks the automatic normalization for regular components. So let's say I have a preact component that uses `class` as a prop in my preact-compat app. If I pass the component a className it won't apply it cause it's relying on `class`, not `className`. I think this is valid for preact-compat mode but that's not my decision to make.

Reply:
I wonder if we could always install descriptors for both `class` and `className`, and simply make the corresponding one `enumerable` based on which was specified?

Reply:
That's what I did now :)